schema.graphql

```diff
@@ -10,20 +10,23 @@ type Domain @entity {
   isLocked: Boolean!
   metadata: String
   royaltyAmount: BigInt!
+  timeStamp: BigInt
   events: [DomainEvent!]! @derivedFrom(field: "domain")
 }
 
 interface DomainEvent {
   id: ID!
   domain: Domain!
   blockNumber: Int!
+  timeStamp: BigInt!
```

Comment:
This might be un-necessary since it's possible to convert `blockNumber` to an approximate time, but it's a nice to have so why not?

Please rename `timeStamp` to `timestamp`

Reply:
I camel cased this to avoid any possible naming convention errors like the one that caused me to have to involve Jordan with the transfer entity. I'll change it and make sure it works but in the future this is why I have been doing this, to preemptively avoid that problem when more than on var share a name.

Reply:
Also, I added the timestamp at the specific request of the front end team